tests/sqlparser_snowflake.rs

```diff
@@ -2400,3 +2400,29 @@ fn parse_use() {
         );
     }
 }
+
+#[test]
+fn parse_view_column_descriptions() {
+    let sql = "CREATE OR REPLACE VIEW v (a COMMENT 'Comment of field') COMMENT = 'Comment of view' AS SELECT a FROM table1 AS o";
+
```

Comment:
Can we also add a roundtrip test with multiple columns (where some have comments and other don't in the same statement), then another that only has column comment but not table-level comment?

Reply:
I updated the unit-test with multiple columns case and removed a view comment.